<launch>
    <arg name='host' default='localhost'/>
    <arg name='port' default='2000'/>
    <arg name='fixed_delta_seconds' default='0.05'/>
    <arg name='role_name' default='ego_vehicle'/>
    <arg name="town" default="Town05"/>
    <arg name="sensor_definition_file" default="$(find telecarla_manual_control)/config/single_sensors.json"/>

    <include file="$(find carla_ros_bridge)/launch/carla_ros_bridge.launch">
        <arg name='host' value='$(arg host)'/>
        <arg name='port' value='$(arg port)'/>
        <arg name="town" value="$(arg town)"/>
        <arg name='fixed_delta_seconds' value='$(arg fixed_delta_seconds)'/>
    </include>

    <include file="$(find carla_spawn_objects)/launch/carla_example_ego_vehicle.launch">
        <arg name="objects_definition_file" value='$(arg sensor_definition_file)'/>
        <arg name='role_name' value='$(arg role_name)'/>
    </include>
</launch>
